### DISCUSSION TITLE
Project Management Unit WebApp

### DISCUSSION THREAD

**Discussion by Mashiane** [pockethost/pockethost#207] [Show and tell]
Hi 

https://sithasodaisypmu.netlify.app/

This is developed using DaisyUI (TailwindCSS components).

This webapp features a couple of modules.
SignIn
SignUp
ForgotPasword
Dashboard
Projects
Events
Events Calendar
Clients
Collaborate (Chat)
Departments
People
Tasks
Tasks Gantt Chart

**Comment by benallfree** [comment]
Please add this to #180.
